src/metrics/family.rs

```diff
@@ -97,7 +97,7 @@ use std::sync::{Arc, RwLock, RwLockReadGuard};
 /// # assert_eq!(expected, String::from_utf8(buffer).unwrap());
 /// ```
 // TODO: Consider exposing hash algorithm.
-pub struct Family<S, M> {
+pub struct Family<S, M, C: MetricConstructor<M> = fn() -> M> {
```

Comment:
```suggestion
pub struct Family<S, M, C = fn() -> M> {
```
Is this bound needed?